extensions/pom.xml

```diff
@@ -198,6 +198,9 @@
 
         <module>awt</module>
 
+       <!-- JMS -->
+       <module>jms-spi</module>
+
```

Comment:
Indentation is off. It should be 4 spaces.

Reply:
Done.